src/cpp/src/llm_pipeline.cpp

```diff
@@ -598,6 +632,36 @@ ov::genai::LLMPipeline::LLMPipeline(
     m_pimpl->m_load_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(stop_time - start_time).count();
 }
 
+ov::genai::LLMPipeline::LLMPipeline(
+    const std::string& model_str,
+    const ov::Tensor& weights_tensor,
+    const ov::genai::Tokenizer& tokenizer,
+    const std::string& device,
+    const ov::AnyMap& config,
+    const ov::genai::GenerationConfig& generation_config
+){
+    auto start_time = std::chrono::steady_clock::now();
+    if (config.find(ov::genai::scheduler_config.name()) != config.end()) {
+        // TODO: check why compiled model does not have scheduler config
```

Comment:
scheduler config is required for CB logic, not for the compiled model.
Or maybe I don't understand this TODO

Reply:
You answered to the question from todo. I will remove this comment.

I've just written there not to forget, because i got suspicious that for this ctor scheduler config is not used.